File: app/src/main/java/com/foxek/simpletimer/ui/SplashActivity.java
package com.foxek.simpletimer.ui;

import android.content.Intent;
import android.os.Bundle;

import com.foxek.simpletimer.ui.workout.WorkoutActivity;

import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, WorkoutActivity.class);
        startActivity(intent);
        finish();
    }

}

File: app/src/main/java/com/foxek/simpletimer/ui/SplashActivity.kt
package com.foxek.simpletimer.ui

import android.content.Intent
import android.os.Bundle

import com.foxek.simpletimer.ui.workout.WorkoutActivity

import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, WorkoutActivity::class.java)
        startActivity(intent)
        finish()
    }

}
